cartographer/io/hybrid_grid_points_processor.cc

```diff
@@ -39,8 +39,12 @@ HybridGridPointsProcessor::FromDictionary(
 }
 
 void HybridGridPointsProcessor::Process(std::unique_ptr<PointsBatch> batch) {
-  range_data_inserter_.Insert({batch->origin, batch->points, {}},
-                              &hybrid_grid_);
+  // TODO(gaschler): Move this to a helper function.
+  sensor::PointCloud points_time(batch->points.size(), Eigen::Vector4f::Zero());
```

Comment:
RangeDataInserter should only deal with a PointCloud, not with a TimedPointCloud anymore.

Reply:
Done